estela-api/api/views/stats.py

```diff
@@ -201,110 +199,75 @@ def summarize_stats_results(
                 )
         return stats_results
 
-    def parse_jobs_stats(
-        self, stats_ids: List[str], stats_set: List[dict]
-    ) -> GetJobsStatsSerializer:
-        reformatted_stats_set: dict = {stat["_id"]: stat for stat in stats_set}
-        jobs_stats_results: List[dict] = []
-
-        for stat_id in stats_ids:
-            ids = findall(r"\d+", stat_id)
-            spider_id, job_id = int(ids[0]), int(ids[1])
-            job_stat_result: dict = {"jid": job_id, "spider": spider_id}
-            stats: Union[dict, None] = reformatted_stats_set.get(stat_id)
-            if isinstance(stats, dict):
-                job_stat_result["stats"] = {}
-                job_stat_result["stats"]["items_count"] = stats.get(
-                    self.stats_mapping["items_count"], 0
-                )
+    def parse_jobs_stats(self, stats_set: List[dict]) -> dict:
+        stats_results = defaultdict(lambda: defaultdict(int))
+        stats_results.default_factory = lambda: {
```

Comment:
This code is repeated in summarize function

Reply:
Ok I added common dictionaries to initialize body responses